neard/src/main.rs

```diff
@@ -1,25 +1,41 @@
-use std::convert::TryInto;
+mod init;
+mod run;
+mod testnet;
+
 use std::env;
 use std::fs;
 use std::io;
 use std::path::Path;
 
-use clap::{crate_version, App, AppSettings, Arg, SubCommand};
+use clap::{crate_version, AppSettings, Clap};
 #[cfg(feature = "adversarial")]
 use tracing::error;
 use tracing::info;
 use tracing_subscriber::filter::LevelFilter;
 use tracing_subscriber::EnvFilter;
 
-use actix;
 use git_version::git_version;
 use near_performance_metrics;
 use near_primitives::version::{Version, DB_VERSION, PROTOCOL_VERSION};
 #[cfg(feature = "memory_stats")]
 use near_rust_allocator_proxy::allocator::MyAllocator;
-use nearcore::config::{init_testnet_configs, load_config_without_genesis_records};
-use nearcore::genesis_validate::validate_genesis;
-use nearcore::{get_default_home, get_store_path, init_configs, start_with_config};
+use nearcore::{get_default_home, get_store_path};
+
+#[macro_use]
+extern crate lazy_static;
+
+lazy_static! {
+    static ref NEARD_VERSION: Version = Version {
+        version: crate_version!().to_string(),
+        build: git_version!(fallback = "unknown").to_string(),
+    };
+    static ref NEARD_VERSION_STRING: String = {
+        format!(
+            "{} (build {}) (protocol {}) (db {})",
+            NEARD_VERSION.version, NEARD_VERSION.build, PROTOCOL_VERSION, DB_VERSION
+        )
+    };
+}
```

Comment:
It's a shame that clap's API forces us to use lazy static for a version, but I don't see a good solution here.